packages/gafl-webapp-service/src/pages/terms-and-conditions/route.js

```diff
@@ -1,11 +1,12 @@
 import pageRoute from '../../routes/page-route.js'
 import * as mappings from '../../processors/mapping-constants.js'
+import { getTrackingProductDetailsFromTransaction } from '../../processors/analytics.js'
 import Joi from '@hapi/joi'
 import { TERMS_AND_CONDITIONS, CONTROLLER, CONTACT_SUMMARY, LICENCE_SUMMARY } from '../../uri.js'
 
 import GetDataRedirect from '../../handlers/get-data-redirect.js'
 
-const getData = async request => {
+export const getData = async request => {
```

Comment:
Same question - why the export

Reply:
Same answer. Whilst it might be possible to extract it from the default export, it's easier and more transparent to just export it and test it as a unit.

Reply:
yes sure